### PR TITLE
fix image creation with local debian packages

### PR DESCRIPTION
**Pull request by lorsatti**
Related to #417.

### CODE REVIEW

**Comment by vania-pooh** [comment]
@lorsatti sorry for the delay. Being on vacation and will check in detail during next days.

**Comment by lorsatti** [comment]
Np. Enjoy your vacation!

File: build/chrome.go [hunk 1]
```diff
@@ -40,7 +40,11 @@ func (c *Chrome) Build() error {
 
 	if pkgSrcPath != "" {
 		srcDir = "chrome/local"
-		pkgDestPath := filepath.Join(devDestDir, "google-chrome.deb")
+		err := os.MkdirAll(devDestDir+"/"+srcDir, 0755)
```
Comment (vania-pooh) [inline build/chrome.go:43]: Should use `filepath.Join` to make file path separator platform-independent. Just declare `pkgDestDir := filepath.Join(devDestDir, srcDir)` and also use this value in `pkgDestPath` initialization.

File: build/edge.go [hunk 2]
```diff
@@ -33,7 +33,11 @@ func (c *Edge) Build() error {
 
 	if pkgSrcPath != "" {
 		srcDir = "edge/local"
-		pkgDestPath := filepath.Join(devDestDir, "microsoft-edge.deb")
+		err := os.MkdirAll(devDestDir+"/"+srcDir, 0755)
```
Comment (vania-pooh) [inline build/edge.go:36]: Use `filepath.Join`.

File: build/firefox.go [hunk 3]
```diff
@@ -40,7 +40,11 @@ func (c *Firefox) Build() error {
 
 	if pkgSrcPath != "" {
 		devSrcDir = "firefox/local"
-		pkgDestPath := filepath.Join(devDestDir, "firefox.deb")
+		err := os.MkdirAll(devDestDir+"/"+devSrcDir, 0755)
```
Comment (vania-pooh) [inline build/firefox.go:43]: Use `filepath.Join`.

File: build/opera.go [hunk 4]
```diff
@@ -31,7 +31,11 @@ func (o *Opera) Build() error {
 
 	if pkgSrcPath != "" {
 		srcDir = "opera/local"
-		pkgDestPath := filepath.Join(devDestDir, "opera.deb")
+		err := os.MkdirAll(devDestDir+"/"+srcDir, 0755)
```
Comment (vania-pooh) [inline build/opera.go:34]: Use `filepath.Join`.

File: build/yandex.go [hunk 5]
```diff
@@ -31,7 +31,11 @@ func (yb *YandexBrowser) Build() error {
 
 	if pkgSrcPath != "" {
 		srcDir = "yandex/local"
-		pkgDestPath := filepath.Join(devDestDir, "yandex-browser.deb")
+		err := os.MkdirAll(devDestDir+"/"+srcDir, 0755)
```
Comment (vania-pooh) [inline build/yandex.go:34]: Use `filepath.Join`.

**Review by vania-pooh — CHANGES_REQUESTED** [review]
Sorry that is took so much time to review. Need to use `filepath.Join` everywhere instead of `/`. See my first comment. All the rest seems to be ok.

**Review by vania-pooh — APPROVED** [review]

**Comment by vania-pooh** [comment]
@lorsatti thank you for contribution!

**Pull request merged**